style.css

```diff
@@ -80,3 +80,9 @@ h2 ~ .badge-text-container {
   color: var(--OFFWHITE);
   padding: 0 0.5em;
 }
+
+.cards-container {
```

Comment:
are you using this class anywhere? if not then you can remove it

Reply:
yes i will remove this.